### PR TITLE
refactor(catalog): storybook for each package & composed storybook

### LINKED ISSUE #22: Integrate Storybook lerna
Since we are moving to deploy our packages with lerna we should have a unified storybook for all the components.

### PR DESCRIPTION
**Pull request by rannyeli**
There is a problem with the composed storybook, I'm waiting for support from `storybookjs`.
related issue storybookjs/storybook#11892

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    '@storybook/addon-actions',
+    '@storybook/addon-knobs',
+    '@storybook/addon-links',
+    '@storybook/addon-docs',
+    'storybook-dark-mode'
+  ],
+  refs: {
+    react_components: { 
+      title: "React Components", 
+      url: "http://localhost:9010"
```
Comment (alebinson) [inline .storybook/main.js:14]: Do we publish story books? 
```http://localhost:9010``` is for development

File: .storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    react_ui_components: { 
+      title: "React UI Components", 
+      url: "http://localhost:9011"
```
Comment (alebinson) [inline .storybook/main.js:18]: Do we publish story books? 
```http://localhost:9010``` is for development

File: README.md [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
 
 The repo uses [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) to link dependencies, and to enable running `yarn install` once.
 
 **You should use yarn instead of npm.**
 
 
 ## Projects
-* **mc-conflict-ui** - UI app for resolving conflicts.
-* **mc-react-components** - Component library for shared components.
+* **demoapp-react** - UI app as a demo for using react components.
```
Comment (alebinson) [inline README.md]: it's an old code, merge from master is needed

**Review by alebinson — CHANGES_REQUESTED** [review]

File: .storybook/main.js (hunk 1 above)
Comment (rannyeli) [reply .storybook/main.js:14]: we will publish them in the future, for now it's only for development

File: README.md (hunk 3 above)
Comment (rannyeli) [reply README.md]: done

**Review by alebinson — APPROVED** [review]
Well done

**Pull request merged**